cmd/placement/README.md

```diff
@@ -38,6 +38,25 @@ In the `root:compute` workspace, create Location `foo`:
 kubectl create -f config/samples/location_foo.yaml
 ```
 
+Note: in the tested kcp [snapshot](https://github.com/kcp-dev/kcp/tree/4506fdc064060b3fe82e1082533f9798b36ba7a5), the `kubectl-workspace` binary prints some hash-like strings for the workspace names.
+```
+👉 edge-mc $ kubectl ws root
+Current workspace is "root".
+👉 edge-mc $ kubectl ws tree
+.
+└── root
+    ├── 2uctu097ej9hv9kv
+    └── 31xvmeuhuk8tykwy
+        └── 1ssldk2xdtia9an1
+```
+However, we can still access the workspaces by their human-readable names, as long as we know the names and their structure.
+```
+👉 edge-mc $ kubectl ws root:compute
+Current workspace is "root:compute".
+👉 edge-mc $ kubectl ws root:my-org:dev
+Current workspace is "root:my-org:dev".
```

Comment:
But a typical user's workspace is named differently.  A typical user's workspace would be entered by `kubectl ws \
``.

Reply:
OK. Tried to handle this in force-push https://github.com/kcp-dev/edge-mc/commit/a56946546957d1038255d6a6178c52e4a64b8ca3.